src/scripts/router.js

```diff
@@ -3,31 +3,45 @@ import { renderer } from './renderer';
 import { camera } from './renderer/camera';
 import { PlanetsScene } from './scenes/planets';
 import { XrScene } from './scenes/xr-scene';
-
-// update scene when page loaded
-navigateToScene(window.location.pathname);
+import { PendulumScene } from './scenes/pendulums';
 
 /**
  * @type {XrScene}
  */
 let currentScene;
+const SavedStates = {};
+
+const Routes = {
+	get '/' () {
+		return new HomeScene(renderer, camera);
+	},
+	get '/planets' () {
+		return new PlanetsScene(renderer, camera);
+	},
+	get '/pendulums' () {
+		if (!this._cached_pendulum) {
```

Comment:
Is there a reason we're caching the pendulum scene? I thought we were only going to cache the home scene (if any)

Reply:
Cool way to do this though!

Reply:
Sorry, I didn't intend to have the temp commit in this merge to begin with.  We could probably use something similar to cache the home room, though.